### DISCUSSION TITLE
Unable to run app with adaptive icon

### DISCUSSION THREAD

**Discussion by Brianggalvez** [appcelerator/titanium_mobile#13201] [Q&A]
After setting up adaptive icons in a brand new android app with the latest SDK (10.1.1.GA) The app does not start and the console outputs the following 

`Unable to start activity ComponentInfo{}: android.content.res.Resources$NotFoundException: drawable/titanium_splash_adaptive_icon_background with resource ID #0x7f07009b`

**Comment by jquick-axway** [comment]
Which `res` folder are you putting your adaptive icon under? (aka: round icon)
Because the error message indicates that the Android OS failed to find the adaptive icon under the app's `res` folders.

Normally you would put it and its foreground/background drawables under the `res/mipmap-*` folders as shown in the example below. Just look for the `ic_launcher_round` files.
https://github.com/appcelerator/hyperloop-examples/tree/master/app/platform/android/res
https://github.com/appcelerator/hyperloop-examples/blob/master/tiapp.xml#L64

Putting it under a `drawable` folder should work too, but I think the real problem is the OS is failing to find it.
